bot/cogs/moderation/scheduler.py

```diff
@@ -174,12 +174,15 @@ async def apply_infraction(
             dm_result = f"{constants.Emojis.failmail} "
             log.trace(f"Deleted infraction {infraction['id']} from database because applying infraction failed.")
             await self.bot.api_client.delete(f"bot/infractions/{infraction['id']}")
+            infr_message = ""
+        else:
+            infr_message = f"**{infr_type}** to {user.mention}{expiry_msg}{end_msg}"
 
         # Send a confirmation message to the invoking context.
         log.trace(f"Sending infraction #{id_} confirmation message.")
         await ctx.send(
             f"{dm_result}{confirm_msg} "
-            f"{f'**{infr_type}** to {user.mention}{expiry_msg}{end_msg}' if not failed else ''}."
+            f"{infr_message}."
         )
```

Comment:
Because it's now a lot shorter, it should all be on one line instead of being broken up.

Reply:
Resolved in e236113.

Reply:
The space should also be moved to the `infr_message` rather than here. Otherwise, there will be a space before the full stop when `failed == True`.

Reply:
Space moved in f1f2c48.